backend/mem/test_files/foo.txt

```diff
@@ -0,0 +1 @@
+Hello World!
```

Comment:
I can't see anywhere where this file is used.  Even if you did have an os File, it would be preferable you set it up as needed as part of your test (and removed it after) in a temp are (ie, using ioutil.TempFile(), or ioutil.TempDir())